export/Dockerfile

```diff
@@ -1,3 +1,21 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.15.14 AS crane-builder
+RUN git clone https://github.com/konveyor/crane $APP_ROOT/src/github.com/konveyor/crane
+WORKDIR $APP_ROOT/src/github.com/konveyor/crane
+## Workaround go 1.15 builder. Remove this after go-toolset 1.16 is available
+RUN sed -i 's/\"os\"/"os"\n\t"io\/ioutil"/g' cmd/apply/apply.go
+RUN sed -i 's/os\.ReadFile/ioutil.ReadFile/g' cmd/apply/apply.go
+RUN sed -i 's/go 1.16/go 1.15/g' go.mod
+## End Workaround go 1.15 builder.
```

Comment:
I'm not fond of doing this, but I haven't come across a 1.16 builder that is available upstream and not on docker.io (where we've met frustration due to rate limiting) yet.

@shawn-hurley @alaypatel07 I would advocate that crane(-lib) be mindful of the availability of images before ratcheting up requirements in the future.

Reply:
Just to make sure that this isn't lost, but the requirement on this version of golang also comes from the upstream kube libraries that we are using IIRC. I think that if this is a problem, then we need to discuss with internal teams about how we can get these builder images faster and not that we must use older versions of libraries. 

I agree that we need to be mindful of this externality though, and that we need to consider how to make this supported in the future, before the next golang version bump will/should make sure that it does not impact the builds. WDYT?

Reply:
Builds appear to complete fine just altering thes few lines, so I don't think there is currently anything in the dependencies forcing 1.16, or at least I don't understand how that's coming into play with requiring 1.16.

Recently completed Quay.io build: https://quay.io/repository/konveyor/gitops-primer-export/build/1161c991-0f4c-4fab-9306-4acc4db31c09